android/pytorch_android/src/main/java/org/pytorch/AndroidUtils.java

```diff
@@ -0,0 +1,16 @@
+package org.pytorch;
+
+import android.content.res.AssetManager;
+
+import com.facebook.soloader.nativeloader.NativeLoader;
+import com.facebook.soloader.nativeloader.SystemDelegate;
+
+public final class AndroidUtils {
+
+  public static Module loadModuleFromAsset(final String assetName, final AssetManager assetManager) {
```

Comment:
I would reverse the arguments and comment that this is meant for tests and demos and that production apps should strongly consider distributing models out-of-band and loading from disk.